hack/deploy_csv.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   namespace: openshift-marketplace
 spec:
   configMap: codeready-toolchain-saas-host-operator
-  displayName: Codeready Toolchain Saas Host Operator
+  displayName: codeready toolchain saas host operator
```

Comment:
This look like another incompatibility of mac :-/

Reply:
@xcoulon Could you please replace these two lines in api olm_catalog.sh script:
https://github.com/codeready-toolchain/api/blob/master/scripts/olm-catalog.sh#L69-L70
```
LOWERCASE=($(echo ${NAME} | tr '-' ' '))
DISPLAYNAME=${LOWERCASE[*]^}
```
by only one line:
```
DISPLAYNAME=$(echo ${NAME} | tr '-' ' ' | sed -e 's/\b\(.\)/\u\1/g')
```
and try to regenerate again?

Reply:
that did not change anything in the generated CRD:
```
$ cat hack/deploy_csv.yaml
# This file was autogenerated by github.com/codeready-toolchain/api/olm-catalog.sh'
---
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: codeready-toolchain-saas-host-operator
  namespace: openshift-marketplace
spec:
  configMap: codeready-toolchain-saas-host-operator
  displayName: codeready toolchain saas host operator
  publisher: Red Hat
  sourceType: internal
---
...
```

Reply:
just for information - this was the correct way:
```
DISPLAYNAME=$(echo ${NAME} | tr '-' ' ' | awk '{for (i=1;i<=NF;i++) $i=toupper(substr($i,1,1)) substr($i,2)} 1')
```